ghost/core/core/server/api/endpoints/posts-public.js

```diff
@@ -31,6 +31,34 @@ module.exports = {
             cacheInvalidate: false
         },
         cache: postsPublicService.api?.cache,
+        generateCacheKey(frame) {
+            const stringify = require('json-stable-stringify');
+            const keyData = {
+                include: frame.options?.include?.split(',').sort(),
+                filter: frame.options?.filter,
+                fields: frame.options?.fields?.split(',').sort(),
+                formats: frame.options?.formats?.split(',').sort(),
+                limit: frame.options?.limit,
+                order: frame.options?.order,
+                page: frame.options?.page || 1,
```

Comment:
Interesting one. Should we consider setting a default for the `options.page` on some other layer to avoid the `||` condition here? 🤔

Reply:
Potentially yeah, but it's tricky because this gets run before most other layers.

Reply:
It doesn't really work unless we shift lots of things round, and start running serialisation before cache key generation. Leaving as-is for now